src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -317,6 +317,11 @@ When it appears in the middle of a sentence, "lint" is not capitalized.
   <data name="XA0124" xml:space="preserve">
     <value>Interpreter is not supported by the x86 ABI</value>
   </data>
+  <data name="XA0125" xml:space="preserve">
+    <value>'{0}' is using a deprecated symbol format. Use 'DebugType=portable' to use the newer, cross-platform symbol format.</value>
+    <comment>The following are literal names and should not be translated: DebugType=portable.
+{0} - The file name of a deprecated symbol file</comment>
+  </data>
```

Comment:
To align with the more verbose way to refer to MSBuild properties started in https://github.com/xamarin/xamarin-android/commit/91022a97214bc6611e61354f723e3b47a0b119e0, I'll suggest:

```xml
  <data name="XA0125" xml:space="preserve">
    <value>'{0}' is using a deprecated symbol format. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform symbol format.</value>
    <comment>The following are literal names and should not be translated: 'DebugType', 'portable'
The capitalized word "Portable" that appears earlier in the message is plain text and should be translated, but the lowercase word "portable" later in the message is a literal value and should not be translated.
{0} - The file name of a deprecated symbol file</comment>
  </data>
```

One other, possibly silly adjustment could be to change "symbol format" to "debug information level" to align with the terminology for this particular MSBuild property on https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2019:

```xml
  <data name="XA0125" xml:space="preserve">
    <value>'{0}' is using a deprecated debug information level. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform debug information level.</value>
    <comment>The following are literal names and should not be translated: 'DebugType', 'portable'
The capitalized word "Portable" that appears earlier in the message is plain text and should be translated, but the lowercase word "portable" later in the message is a literal value and should not be translated.
{0} - The file name of a deprecated symbol file</comment>
  </data>
```

_Reference info:_

In Visual Studio, the property is **Build > Advanced \[button\] > Debugging information** in the project property pages.

In Visual Studio for Mac, the property is **Build > Compiler > Debug information** in the project property pages.

"Debugging information" is hopefully a close enough compromise to cover both.